### PR TITLE
Fixes #34200 - use pf4 pagination from core

### PR DESCRIPTION
**Pull request by MariaAga**
Should be tested with theforeman/foreman#8990

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
The core PR is in, shall we undraft this? :)

**Review by adamruzicka — COMMENTED** [review]
Seems to work well. Since it requires something in foreman core, could you bump the required foreman version in engine.rb?

@Ron-Lavi could we get your blessing on this?

**Comment by MariaAga** [comment]
Thanks! thank would foreman 3.2.0, right?

**Comment by adamruzicka** [comment]
Correct

File: webpack/ForemanTasks/Components/TasksTable/TasksTable.js [hunk 1]
```diff
@@ -82,7 +81,7 @@ const TasksTable = ({
     return <span>{__('No Tasks')}</span>;
   }
 
-  const changeTablePage = ({ page, perPage }) => {
+  const changeTablePage = ({ page, per_page: perPage }) => {
```
Comment (Ron-Lavi) [inline webpack/ForemanTasks/Components/TasksTable/TasksTable.js:84]: I guess this isn't need anymore as core already take cares of updating the url

File: webpack/ForemanTasks/Components/TasksTable/TasksTable.js [hunk 2]
```diff
@@ -18,7 +18,6 @@ const TasksTable = ({
   results,
   history,
   itemCount,
-  pagination,
```
Comment (Ron-Lavi) [inline webpack/ForemanTasks/Components/TasksTable/TasksTable.js:21]: can we also remove `pagination` and its selector from the `index.js` wrapper?

**Review by Ron-Lavi — COMMENTED** [review]
Thanks @MariaAga for doing this !

File: webpack/ForemanTasks/Components/TasksTable/TasksTable.js (hunk 1 above)
Comment (MariaAga) [reply webpack/ForemanTasks/Components/TasksTable/TasksTable.js]: yup, removed

File: webpack/ForemanTasks/Components/TasksTable/TasksTable.js (hunk 2 above)
Comment (MariaAga) [reply webpack/ForemanTasks/Components/TasksTable/TasksTable.js:21]: The component still needs the perpage so change the selector from pagination to perpage (for selectAll functionality)

**Review by adamruzicka — APPROVED** [review]
Works nicely

**Comment by adamruzicka** [comment]
Thank you @MariaAga & @Ron-Lavi !

**Pull request merged**